### DISCUSSION TITLE
Chapter Title on Android Media Player

### DISCUSSION THREAD

**Discussion by erikmatijevich1** [advplyr/audiobookshelf-app#1216] [General]
Apologies if this is a duplicate post or if it's just user error on my end. I've been trying to find an answer to this but to no avail.

When playing an audiobook, only the book title is displayed on the lockscreen / notification bar / Android auto.

![example](https://github.com/advplyr/audiobookshelf-app/assets/59978918/1c683a86-558e-4dee-9c53-2b11be9b47fa)

I would absolutely love it if I would be able to see the chapter title there instead of the book title. I'm not sure if this is even an audiobookshelf issue, and might just be the media player on my phone? If that's the case, does anyone know how I could replace the default media player?

My phone is a Samsung running Android 14, One UI 6.1

Thanks!

**Comment by springsunx** [comment]
#1282

**Comment by ZLoth** [comment]
Instead of the author name, how about the chapter name?
